log/internal/rollbar/rollbar.go

```diff
@@ -26,13 +26,19 @@ func Setup(l *zap.SugaredLogger, service string) func() {
 
 	env := os.Getenv("PACKET_ENV")
 	if env == "" {
-		log.Panicw("required envvar is unset", "envvar", "PACKET_ENV")
+		env = os.Getenv("PKG_ENV")
```

Comment:
we could just do `env.Get("PKG_ENV", env.Get("PACKET_ENV"))` instead

Reply:
Also, I think something more generic would be good here like maybe just `Env`?

Reply:
If we made this just ENV...   would you want PKG_VERSION to be just 'VERSION' (presuming we didn't make your log.Init change at this time)

Reply:
neither, don't see the point of adding a new variable if we want to go away from *soon*.

Reply:
but if we had to then I guess `VERSION`

Reply:
The task as it was assigned to me had 'backwards compatibility' with the old env vars as a piece of acceptance criteria...